### PR TITLE
Add introduction sentence to visualizing repository data with graphs

### LINKED ISSUE #6320: Intro for "Visualizing repository data with graphs" category
### What article on docs.github.com is affected?

The "[Visualizing repository data with graphs](https://docs.github.com/en/github/visualizing-repository-data-with-graphs)" category of articles.

### What part(s) of the article would you like to see updated?

Write intros to help orient users to the content in the category. Intros should be one sentence long. For more information on writing intros, see:
- [Information about categories](https://github.com/github/docs/blob/main/contributing/content-model.md#category) in the GitHub Docs content model
- [Information about writing intros](https://github.com/github/docs/blob/main/contributing/content-model.md#intro) in the GitHub docs content model
- [Information about the `intro` frontmatter property](https://github.com/github/docs/tree/main/content#intro) in the GitHub Docs content directory README
- Make sure your intro is general enough to cover the content in each version of the page (use the version picker in the upper-right corner of the page to see what’s included on the page for different versions of GitHub)

### Additional information

This issue is for the Campus Experts + Docs program and is not available for contributors outside the program to work on.

### PR DESCRIPTION
**Pull request by Perkles**
### Why:

Closes #6320 

### What's being changed:

Add a sentence long introduction to [Visualizing repository data with graphs](https://docs.github.com/en/github/visualizing-repository-data-with-graphs).

### Check off the following:

- [X] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks so much for opening a PR @Perkles! I'll get this triaged for review ⚡

File: content/github/visualizing-repository-data-with-graphs/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Visualizing repository data with graphs
+intro: You can visualize data effectively using graphs.
```
Comment (felicitymay) [inline content/github/visualizing-repository-data-with-graphs/index.md:3]: Hi @Perkles - thanks for adding an introduction to this category.

When we add an introduction, we try to use different terminology from the article title. I can see you've done this in earlier pull requests, but maybe this one was more difficult.

Could you update this introduction to use other terms? For example, maybe some of these might work: display, view, information, charts.

**Review by pipecc — DISMISSED** [review]

File: content/github/visualizing-repository-data-with-graphs/index.md (hunk 1 above)
Comment (Perkles) [reply content/github/visualizing-repository-data-with-graphs/index.md]: Hello @felicitymay 👋 
Sure! I've rephrased the intro almost entirely. 
Let me know what you think about it.

File: content/github/visualizing-repository-data-with-graphs/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Visualizing repository data with graphs
+intro: You can use a variety of graphs to better understand a repository.
```
Comment (felicitymay) [inline content/github/visualizing-repository-data-with-graphs/index.md:3]: Thanks for the update and for expanding on the original introduction. ✨ 

I'm going to make a small change for clarity:

```suggestion
intro: You can use a variety of graphs to gain a better understanding of your repositories.
```

**Review by felicitymay — APPROVED** [review]

**Pull request merged**